src/js/components/atoms/content-link/content-link.ts

```diff
@@ -0,0 +1,31 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { IContentLinkProps } from './content-link.models';
+@Component({
+    name: 'content-link',
+    template: require('./content-link.html'),
+})
+
+export default class ContentLink extends Vue implements IContentLinkProps {
+    @Prop({ type: String, default: '' }) block!: string;
+    @Prop({ type: String, default: '' }) mod!: string;
+    @Prop({ type: String, default: '' }) icon!: string;
+    @Prop({ type: String, default: '' }) href!: string;
+
```

Comment:
Because you have "!" on the props means that is required right? Soo no default value is needed here.